### DISCUSSION TITLE
OrderList drag/drop functionality & keyboard input

### DISCUSSION THREAD

**Discussion by hmcd101** [primefaces/community#1262] [PrimeReact]
Hi Everyone. I have a project that is currently using the [OrderList](https://primereact.org/orderlist/#dragdrop) component, and I was hoping to use the drag&drop feature. 

However, it does not seem to be working. I can see the transparent component being dragged, but it is not triggering anything when dropped somewhere else in the OrderList. 

I checked the [example code](https://primereact.org/orderlist/#dragdrop) to see if I missed something, but noticed the same behaviour there. 

Is there a known issue with the OrderList, or am I perhaps just missing something?

Any advice would be appreciated.

**Comment by melloware** [comment]
@hmcd101 i would report a bug.  Most likely something broke when they added the new keyboard accessibility changes.
